data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>Image isn't rendering.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Hmm, I have no idea about this. It shows up in Jupyterlab and in GitHub when I'm looking at it here - is it maybe a ReviewNB thing? https://github.com/leahecole/python-docs-samples/blob/73d1c20f60363f99d395982bfa482b89ef161116/data-science-onramp/feature-engineering/feature_engineering.ipynb</p>

Reply:
<p>Interesting, I can see it there! All good :) </p>